Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -312,36 +185,27 @@ open class LottieAnimationView: LottieAnimationViewBase {
     loopMode: LottieLoopMode? = nil,
     completion: LottieCompletionBlock? = nil)
   {
-    guard let from = animation?.markerMap?[marker] else {
-      return
-    }
-
-    play(
-      fromFrame: from.frameTime,
-      toFrame: from.frameTime + from.durationFrameTime,
-      loopMode: loopMode,
-      completion: completion)
+    self.lottieAnimationLayer.play(marker: marker, loopMode: loopMode, completion: completion)
   }
 
   /// Stops the animation and resets the view to its start frame.
   ///
   /// The completion closure will be called with `false`
   open func stop() {
-    removeCurrentAnimation()
-    currentFrame = 0
+    self.lottieAnimationLayer.stop()
```

Comment:
Airbnb's Swift Style Guide says to use implicit self where possible, so all of these should be:

```suggestion
    lottieAnimationLayer.stop()
```